src/main/scala/com/wavesplatform/history/HistoryWriterImpl.scala

```diff
@@ -85,11 +114,22 @@ class HistoryWriterImpl private(file: Option[File], val synchronizationToken: Re
   }
 
   override def close(): Unit = db.close()
+
+  override def status(feature: Short): FeatureStatus = read { implicit lock =>
+    val h = height()
+    val lastApprovalHeight = h - h % FeatureApprovalBlocksCount
+    Option(featuresState().get(lastApprovalHeight)).map { m =>
+      val byte: Byte = m.getOrElse(feature, FeatureStatus.Defined.status)
+      FeatureStatus(byte)
+    }.getOrElse(FeatureStatus.Defined)
```

Comment:
Sooo if anything goest wrong with persisting features/rollbacks, we default to `Defined`.
I don't think it represents what really happens, e.g. if I ask the status of feature 862, I guess it should be None, not Defined

Reply:
It should be Defined, because it's in spec